src/modules/songs/components/SongDetailsPage/SongDetailsPageHeader/SongDetailsPageHeader.tsx

```
@@ -21,6 +24,40 @@ export const SongDetailsPageHeader: React.FC<SongDetailsPageHeaderProps> = ({
 }) => {
   const [isEditingName, setIsEditingName] = useState<boolean>(false);
 
+  const router = useRouter();
+
+  const updateSongFavoriteStatusMutation =
+    api.song.updateFavoriteStatus.useMutation();
+
+  const updateSongFavoriteStatus = () => {
+    const toastId = toast.loading(
+      `${!!song.favoritedAt ? "Unfavoriting" : "Favoriting"} song...`,
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)